src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java

```diff
@@ -223,6 +231,14 @@ public void studyBurstIntervalCannotBeNull() {
         assertValidatorMessage(INSTANCE, schedule, STUDY_BURSTS_FIELD + "[0]." + INTERVAL_FIELD, CANNOT_BE_NULL);
     }
 
+    @Test
+    public void studyBurstIntervalCannotBeStrangPeriod() {
```

Comment:
same as above

Reply:
Fixed, also why did I use the word "strange" and not "invalid" — fixed